NOTICE

```diff
@@ -1,5 +1,46 @@
+--------------------------------
+NOTICE for commons-io:commons-io
+--------------------------------
+
 Apache Commons IO
 Copyright 2002-2019 The Apache Software Foundation
 
 This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
\ No newline at end of file
+The Apache Software Foundation (http://www.apache.org/).
+
+
+-----------------------------------
```

Comment:
Wait a sec, I'm getting confused. We don't need these as we don't bundle anything in the distribution; we don't make any uber JAR here. Hm, I think I should pull these out.